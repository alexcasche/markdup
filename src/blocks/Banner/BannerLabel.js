import styled from 'styled-components'
import { shade } from 'polished'

const BannerLabel = styled.label`
  display: block;
  color: ${props => shade(0.85, props.theme.colors.grey)};
  font-size: 11px;
  text-transform: uppercase;
  letter-spacing: 2px;
`

export default BannerLabel
