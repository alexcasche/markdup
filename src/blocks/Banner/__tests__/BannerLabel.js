import BannerLabel from '../BannerLabel'

describe('BannerLabel Element', () => {
  it('renders correctly', () => {
    const tree = render(
      <BannerLabel for="BannerInput">Workspaces</BannerLabel>
    ).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
