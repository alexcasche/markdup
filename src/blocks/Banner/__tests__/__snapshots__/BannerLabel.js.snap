// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerLabel Element renders correctly 1`] = `
.c0 {
  display: block;
  color: #919398;
  font-size: 11px;
  text-transform: uppercase;
  -webkit-letter-spacing: 2px;
  -moz-letter-spacing: 2px;
  -ms-letter-spacing: 2px;
  letter-spacing: 2px;
}

<label
  className="c0"
>
  Workspaces
</label>
`;
